changelog.d/20211201_145304_aaschaer_scopes_type.rst

```diff
@@ -0,0 +1 @@
+* ClientCredentialsAuthorizer now accepts Union[str, Iterable[str]] as the type for scopes
```

Comment:
```suggestion
* ClientCredentialsAuthorizer now accepts Union[str, Iterable[str]] as the type for scopes (:pr:`NUMBER`)
```

The `:pr:` role should be included here. When it's merged, the github actions workflow will populate it. (Technically, now that the PR is open, we could just write `498` instead of `NUMBER`. But the norm should be, I think, to let github actions do the rewrite.)

Reply:
Ah thanks!